components/brave_rewards/resources/page/promos/index.tsx

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import tapBg from './assets/tap-bg.svg'
+import { StyledLink, StyledInfo } from '../../ui/components/sidebarPromo/style'
+
+const openTapNetwork = () => {
+  window.open('https://brave.tapnetwork.io', '_blank')
+}
+
+export type PromoType = 'tap-network'
+
+export interface Promo {
+  title: string,
+  imagePath: string,
+  copy: JSX.Element,
+  disclaimer?: string,
+  supportedLocales: string[]
+}
+
+export const activePromos = [
+  'tap-network'
+]
+
+export const getPromo = (type: PromoType, getLocale: any) => {
```

Comment:
we have to pass in `getLocale` upon creating the promo, because if we import it in this file, `loadTimeData` won't be defined yet for strings when this is required.